kafka/schemas/airqo-device-measurements.avsc

```diff
@@ -0,0 +1,94 @@
+
+{
+    "namespace": "net.airqo.models",
+    "type": "record",
+    "name": "AirQoDeviceMeasurements",
+    "doc": "http://34.78.78.202:31001/api/v1/data/feeds/transform/recent?channel=1014687",
```

Comment:
- Can we try to put these endpoints in a central config place within this setup? Possible?
- Could we also try to use domain names instead of IPs? And https ones at that.